src/components/GlobalStatusSummary.jsx

```diff
@@ -8,15 +8,29 @@ const statusReducer = (endpointStatuses) => {
   const criticalEndpoints = Object.keys(endpointStatuses).filter(e => endpointStatuses[e].critical);
   const nonCriticalEndpoints = Object.keys(endpointStatuses)
     .filter(e => !endpointStatuses[e].critical);
+
+  const anyFatalOutages = (() => {
+    const appStatus = endpointStatuses.searchworksApplication.status;
+    const solrStatus = endpointStatuses.swSolr.status;
+    const articleStatus = endpointStatuses.ebsco.status;
+
+    if (appStatus === 'outage' || (solrStatus === 'outage' && articleStatus === 'outage')) {
+      return true;
+    }
+    return false;
+  })();
```

Comment:
Not sure if this anonymous function assigned to a constant and invoked inline is the best way to accomplish this sort of thing.

Reply:
Yeah.. perhaps we could abstract class to a POJO? I think what you have could work for now, but that could separate the globabl status logic from the view?

Reply:
I ❤️ POJOs.  BRB, gotta do some refactorin'